symfony/plugins/orangehrmMarketPlacePlugin/lib/dao/MarketplaceDao.php

```diff
@@ -85,6 +107,31 @@ public function installOrRequestAddon($data)
         // @codeCoverageIgnoreEnd
     }
 
+    /**
+     * @param $status
+     * @return bool
+     * @throws DaoException
+     */
+    public function updateStatusRequestAddon($paidAddonNames)
```

Comment:
better to use in a generic format 
changeAddonStatus( string $fromStatus. string $toStatus): bool {
}

Reply:
This is basically to change the status for the paid addons which we are retrieving from the marketplace. So as a param we are sending an array of paid addon names.